tests/integration/blockBrowserAPIS.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Ensure that some browser API are available, to block compat behavior.
+ */
+export default function blockBrowserAPIS() {
```

Comment:
The name of this function and of this file is too generic, and the name doesn't sound really adapted to what it actually does.

Reply:
Also, what is the problem here? Is it to test it on safari?

If it is, I do not see the point of lying about browser support in tests...